cuda_pathfinder/cuda/pathfinder/_dynamic_libs/load_dl_linux.py

```diff
@@ -14,48 +14,40 @@
 
 LIBDL_PATH = ctypes.util.find_library("dl") or "libdl.so.2"
 LIBDL = ctypes.CDLL(LIBDL_PATH)
-LIBDL.dladdr.argtypes = [ctypes.c_void_p, ctypes.c_void_p]
-LIBDL.dladdr.restype = ctypes.c_int
+LIBDL.dlinfo.argtypes = [ctypes.c_void_p, ctypes.c_int, ctypes.c_void_p]
+LIBDL.dlinfo.restype = ctypes.c_int
```

Comment:
I am a bit nervous about us using either `dladdr` or `dlinfo` since none of them is POSIX-compliant. Since `dladdr` has been used, switching to `dlinfo` does not make it more "non-compliant", so it's still a step forward. We just have to see if it runs everywhere...